packages/issuer-sdk-js/README.md

```diff
@@ -9,13 +9,123 @@ Get [our NPM package](https://www.npmjs.com/package/@idos-network/idos-sdk) and
 pnpm add @idos-network/idos-issuer-sdk-js
 ```
 
+
+## Quickstart
+
+Create an issuer config with your secret key. This config will be used to interact with the idOS.
+```js
+import { createIssuerConfig } from "@idos-network/idos-issuer-sdk-js";
+import { Wallet } from "ethers";
+
+const wallet = new Wallet("YOUR_PRIVATE_KEY");
+
+const issuerConfig = await createIssuerConfig({
+  nodeUrl: "https://node.url",
+  secretKey: "YOUR_SECRET_KEY",
+  signer: wallet // Or use a KeyPair from near-api-js
+});
+```
+### Creating a human profile.
+
+In order to create a human profile in idOS, you need to have a wallet associated with the human and a public key for the human.
+Passing ids are optional and will be auto-generated if not provided.
```

Comment:
Since it's an advanced usage scenario, I'd omit this feature to reduce the reader's cognitive load.